app/views/admin/service_announcements/new.html.haml

```diff
@@ -0,0 +1,11 @@
+- content_for :page_title do
+  New Service Announcement
+
+- content_for :body do
+  .ui.two.wide.column
+    .ui.row
+      = render partial: "menu", locals: { count: @count }
```

Comment:
Isn't there's some trick here where the locals get populated automatically from @instance variables?

Reply:
You can pass `object: @count` instead and it'll be available in the partial as `menu` I think? But other than that, not that I've found